new_relic_scripts/support/twilio_helpers.js

```diff
@@ -0,0 +1,72 @@
+const Q = require('q');
+const querystring = require('querystring');
+const request = require('request');
+
+const {
+  TWILIO_SID,
+  TWILIO_TOKEN,
+} = process.env;
+
+const TWILIO_MESSAGES_URL = [
+  'https://api.twilio.com/2010-04-01/Accounts',
+  TWILIO_SID,
+  'Messages.json'
+].join('/');
+
+const getTextMessages = ({ sentAfter, phoneNumber }) => {
+  const deferred = Q.defer();
```

Comment:
This is a weird way to do this, but it is necessary because there's some weirdness around Promises on New Relic: https://discuss.newrelic.com/t/unknownerror-when-mixing-native-promises-and-selenium/51216/2